### ISSUE TITLE
How to use MOSEI dataset to classify 6 discrete emotions?

### ISSUE THREAD

**Issue by qianfan1996** [A2Zadeh/CMU-MultimodalSDK#211]
Hi, I'd like to ask a stupid question... I observe that there are multiple emotions in each segment of CMU-MOSEI dataset. And how to use MOSEI dataset to classify 6 discrete emotions? Thank you in advance.

**Comment by A2Zadeh** [comment]
Hi @qianfan1996. Of course. You can find this information here: https://github.com/A2Zadeh/CMU-MultimodalSDK/tree/master/mmsdk/mmdatasdk/dataset/standard_datasets/CMU_MOSEI

**Comment by qianfan1996** [comment]
Thanks a lot for your quick help. So is it a multilabel multiclass task? Are softmax + crossentropy not allowed? @A2Zadeh

**Comment by A2Zadeh** [comment]
@qianfan1996,

Since emotions are not mutually exclusive, we can have multiple emotions present (e.g. anger and sadness). I suggest using Sigmoid or ReLU.

**Comment by qianfan1996** [comment]
Thanks for your reply, it is very helpful! By the way, I downloaded the word-aligned dataset IEMOCAP, but I don't know the password for the zip file. I've got the permission from USC. Could you help me? @A2Zadeh

**Comment by A2Zadeh** [comment]
@qianfan1996 you can reach out to me via email and I can share the password.
